src/tests/suite/daos_nvme_recovery.c

```diff
@@ -254,11 +254,203 @@ nvme_test_verify_device_stats(void **state)
 	D_FREE(devices);
 }
 
+/* Verify NVMe I/O error and notification*/
```

Comment:
More test description needed I think. "Simulate both an NVMe I/O read and write error. Check error counters in BIO health stats to verify R/W error counts, and also verify I/O error notification on the console output"

Reply:
Fixed